src/stats/src/MetropolisHastingsSG.C

```diff
@@ -787,7 +797,11 @@ MetropolisHastingsSG<P_V,P_M>::alpha(
   const P_V& _lastTKPosition         = m_tk->preComputingPosition(        tkStageIds[inputSize-1]);
   const P_V& _lastBackwardTKPosition = m_tk->preComputingPosition(backwardTKStageIds[inputSize-1]);
 
+  // unsigned int old_stageId = m_tk->set_dr_stage();
```

Comment:
Are these commented-out lines still here on purpose?  If they're supposed to be explanatory of something in the surrounding uncommented code, I'm failing to figure out what.